packages/components/addon/components/hds/table/index.js

```diff
@@ -179,4 +187,97 @@ export default class HdsTableIndexComponent extends Component {
       onSort(this.sortBy, this.sortOrder);
     }
   }
+
+  @action
+  onSelectionAllChange() {
+    // this.setSelectAllCheckboxAriaLabel(this.selectAllCheckbox);
+    this.selectableRows.forEach((row) => {
+      row.checkbox.checked = this.selectAllCheckbox.checked;
```

Comment:
@didoo This isn't changing the `isSelected` tracked value of the row checkbox, so the ariaLabel for the rows don't change when you select or deselect all (using the header checkbox)

So far I haven't figured a fix for this (although I'll keep investigating). It looks like you were trying something previously that you commented out though?

Reply:
It's a tricky one. I have to think about it. 🤔